CMakeLists.txt

```diff
@@ -18,11 +18,51 @@ project(ufs-weather-model
         LANGUAGES C CXX Fortran)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-find_package(MPI REQUIRED)
-find_package(ESMF REQUIRED)
 
+message("")
+message("Setting configuration for $ENV{CMAKE_Platform}")
+message("")
 include(${PROJECT_SOURCE_DIR}/cmake/configure_${CMAKE_Platform}.cmake)
 
+message("32BIT ............ ${32BIT}")
+message("AVX2 ............. ${AVX2}")
+message("CCPP ............. ${CCPP}")
+message("DEBUG ............ ${DEBUG}")
+message("INLINE_POST ...... ${INLINE_POST}")
+message("MULTI_GASES ...... ${MULTI_GASES}")
+message("OPENMP ........... ${OPENMP}")
+message("PARALLEL_NETCDF .. ${PARALLEL_NETCDF}")
+message("QUAD_PRECISION ... ${QUAD_PRECISION}")
+message("REPRO ............ ${REPRO}")
+message("WW3 .............. ${WW3}")
+message("")
+
+get_filename_component (C_COMPILER_NAME ${CMAKE_C_COMPILER} NAME)
+get_filename_component (CXX_COMPILER_NAME ${CMAKE_CXX_COMPILER} NAME)
+get_filename_component (Fortran_COMPILER_NAME ${CMAKE_Fortran_COMPILER} NAME)
+message("C       compiler: ${CMAKE_C_COMPILER_ID} ${CMAKE_C_COMPILER_VERSION} (${C_COMPILER_NAME})")
+message("CXX     compiler: ${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION} (${CXX_COMPILER_NAME})")
+message("Fortran compiler: ${CMAKE_Fortran_COMPILER_ID} ${CMAKE_Fortran_COMPILER_VERSION} (${Fortran_COMPILER_NAME})")
+message("")
+
+include(cmake/${CMAKE_Fortran_COMPILER_ID}.cmake)
+message("")
+
+find_package(NetCDF REQUIRED C Fortran)
+find_package(MPI REQUIRED)
+find_package(ESMF MODULE REQUIRED)
+
+find_package(bacio REQUIRED)
+find_package(nemsio REQUIRED)
+find_package(sp REQUIRED)
+find_package(w3emc REQUIRED)
+find_package(w3nco REQUIRED)
+if(INLINE_POST)
+  find_package(nceppost REQUIRED)
+endif()
+# See https://github.com/NOAA-EMC/NCEPLIBS-nemsio/pull/22
```

Comment:
This was merged. Is a comment to a PR still necessary?  
May be more appropriate to say, why mixed types are being used.

Reply:
Yes. That PR was merged. I added that comment here just to remind us why we need to explicitly define nemsio's dependency on specific versions of w3emc and bacio.